kcharles52/books-invetory
# chore(client): initial client setup

**What does this PR do?**

- Sets up the client-side of the project

**Description of Task to be completed?**

- add react app to the project
- add book list component

**How should this be manually tested?**

- Run the following commands
- `git fetch client-setup`
- `git checkout client-setup`
- `cd client` - change to the client folder
- `yarn start`
- navigate to `localhost:3000`
- A page should be displayed

**Any background context you want to provide?**

- The project only had server-side that was being accessed using the `GraphiQL`

**Screenshots (if appropriate)**

![Screenshot 2019-09-04 at 14 47 38](https://user-images.githubusercontent.com/1590202/64252225-fbb08c80-cf22-11e9-8a0d-b7d876232186.png)


**Questions:**

- None
